src/python/module/smartredis/util.py

```diff
@@ -94,7 +103,7 @@ def init_default(default, init_value, expected_type=None):
         raise TypeError(f"Argument was of type {type(init_value)}, not {expected_type}")
     return init_value
 
-def exception_handler(func):
+def exception_handler(func: t.Callable[_PR, _RT])->TFunc:
```

Comment:
Similar "allias doesn't look like it is being followed" issue here. can we also make this
```py
def exception_handler(func: t.Callable[_PR, _RT]) -> t.Callable[_PR, _RT]:
```